saga/inMemorySagaLog.go

```diff
@@ -0,0 +1,71 @@
+package saga
+
+import "errors"
```

Comment:
Can do!

I have gofmt set up on save to and as part of our build!  Dan was saying for imports it mimics however the file was set up.  If we want enforce this overall we may want to set up a linter, or just do it via code reviews.